src/libstore/ssh-store.hh

```diff
@@ -0,0 +1,40 @@
+#pragma once
```

Comment:
AFAICT, this entire header file can be removed since nothing accesses SSHStore from outside `ssh-store.cc`.